src/native-stack/views/NativeStackView.tsx

```diff
@@ -319,31 +349,36 @@ const RouteView = ({
         });
       }}
       onWillAppear={() => {
+        console.log(`onWillAppear/transitionStart route: ${route.key}`);
```

Comment:
Remove console.logs

Reply:
Will do it just before merging, noted down on my list